src/FastRouteRouter.php

```diff
@@ -326,6 +381,80 @@ private function injectRoute(Route $route)
         }
 
         $this->router->addRoute($methods, $route->getPath(), $route->getPath());
-        $this->routes[$route->getName()] = $route;
+    }
+
+    /**
+     * Get the dispatch data either from cache or freshly generated by the
+     * FastRoute data generator. If caching is enabled store the freshly
+     * generated data to file.
+     *
+     * @return array
+     */
+    private function getDispatchData()
+    {
+        if ($this->hasCache) {
+            return $this->dispatchData;
+        }
+
+        $dispatchData = $this->router->getData();
+
+        if ($this->cacheEnabled) {
+            $this->cacheDispatchData($dispatchData);
+        }
+
+        return $dispatchData;
+    }
+
+    /**
+     * Load dispatchData from cache
+     *
+     * @return void
+     * @throws Exception\RuntimeException If the cache file contains invalid data
+     */
+    private function loadDispatchData()
+    {
+        if (! is_file($this->cacheFile)) {
```

Comment:
@Ocramius 
Hello, added here https://github.com/zendframework/zend-expressive-fastroute/pull/16/commits/6d145097d6f8635c89a28aeda0616afd3ff9813c

btw, is it possible to make th cs test pass with the anonymous empty-body function in one line (previous commit didn't pass the php-56 wher cs_check is enabled); i have seen it somewhere in other zend components?

kind regards